### ISSUE TITLE
Cannot navigate to buttons inside bookmark balloon using TAB

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#18292]
## 📝 Provide detailed reproduction steps (if any)

1. Open [bookmarks](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/bookmarks.html) demo.
2. Click a bookmark.
3. Press TAB.
### ✔️ Expected result

Focus moves to the `Edit bookmark` button in the balloon.

https://github.com/user-attachments/assets/fe963cd6-8813-4bcd-ba0a-ddf265baad60

### ❌ Actual result

Focus goes outside of the balloon.

https://github.com/user-attachments/assets/6ddfc67f-f970-452f-9387-efdb2db00e2d

### ❓ Possible solution

It used to work some time ago. The `link` balloon still works as expected, so perhaps we can compare the implementations to identify any differences.

## 📃 Other details

* Browser: …
* OS: …
* First affected CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by pszczesniak** [comment]
Probably missing `Tab` handler like it is done in `LinkUI`:

https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-link/src/linkui.ts#L738-L749

**Comment by Mati365** [comment]
I'll take a look.

**Comment by Mati365** [comment]
I discovered that `ESC` is not working for balloon toolbar in bookmarks. It needs to be fixed too.

**Comment by niegowski** [comment]
This is the standard behavior of the widget toolbar repository. Only the link UI has an additional tab and ESC handling, but it is not a widget and implements a custom balloon with the link toolbar. Link UI is special for the backward compatibility of UX.
